terraform/modules/daily_snapshot/service/daily_snapshot.rb

```diff
@@ -17,6 +15,19 @@
 BUCKET = get_and_assert_env_variable 'SNAPSHOT_BUCKET'
 ENDPOINT = get_and_assert_env_variable 'SNAPSHOT_ENDPOINT'
 
+# Query the date of the most recent snapshot.
+def latest_snapshot_date(chain_name = 'calibnet')
+  # We do not support HEAD requests but we _do_ support empty ranges.
+  filename = `curl --remote-name --remote-header-name --write-out "%{filename_effective}" --silent https://forest-archive.chainsafe.dev/latest/#{chain_name}/ -H "Range: bytes=0-0"`
+  # Curl will create a file with a single byte in it. Let's clean it up.
```

Comment:
could we not save it to a file instead?

Reply:
I'd rather not deal with file management in Ruby. Forest is much better at it.

Reply:
And we _do_ need a way to get the date _without_ downloading the file in any case.